files/zh-cn/learn/html/introduction_to_html/html_text_fundamentals/index.md

```diff
@@ -873,9 +873,7 @@ textarea.onkeyup = () => {
 
 ```html
```

Comment:
To avoid formatting by linter tool, can you change the language label here:

````suggestion
```html-nolint
````